src/main/kotlin/com/dansplugins/factionsystem/command/faction/addmember/MfFactionAddMemberCommand.kt

```diff
@@ -0,0 +1,110 @@
+package com.dansplugins.factionsystem.command.faction.addmember
+
+import com.dansplugins.factionsystem.MedievalFactions
+import com.dansplugins.factionsystem.faction.MfFaction
+import com.dansplugins.factionsystem.faction.MfFactionMember
+import com.dansplugins.factionsystem.player.MfPlayer
+import dev.forkhandles.result4k.onFailure
+import org.bukkit.ChatColor
+import org.bukkit.command.Command
+import org.bukkit.command.CommandExecutor
+import org.bukkit.command.CommandSender
+import org.bukkit.command.TabCompleter
+import org.bukkit.entity.Player
+import preponderous.ponder.command.dropFirst
+import java.util.logging.Level
+
+class MfFactionAddMemberCommand(private val plugin: MedievalFactions) : CommandExecutor, TabCompleter {
+
+    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
+        if (!sender.hasPermission("mf.force.addmember")) {
```

Comment:
Do we want to maintain compatibility with MF 4 permissions here?

Reply:
Yeah that would be good. We're now checking the 'mf.force.join' permission in addition to 'mf.force.addmember'.